lib/src/custom_items/algorithms_page.dart

```diff
@@ -1,28 +1,13 @@
 import 'package:flutter/material.dart';
-import 'simulation_card.dart';
-import '../simulations/toothpick.dart';
-import '../simulations/langton_ant.dart';
+import '../simulations/simulations.dart';
```

Comment:
We should follow some consistent pattern, right?

In some places, we have `package` prefix before, however in some parts we don't.

I'd suggest prefixing only the in-built packages with `package`, not the ones we made. What say?